application/helpers/lib_helper.php

```diff
@@ -26,7 +26,7 @@ function err_msg($code) {
     function load_cookie_individual() {
         $CI = get_instance();
         $data = $CI->input->cookie('individual');
-        return json_decode($data, true);
+        return json_decode(stripslashes($data), true);
```

Comment:
I think we should revert this change as why we need `stripslashes` is caused by our other mistakes.